pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/model/IngestUpdate.scala

```diff
@@ -1,3 +1,18 @@
 package uk.ac.wellcome.platform.transformer.mets.model
 
-case class IngestUpdate()
+case class IngestUpdate(space: Space, bag: StubBag)
+
+case class Space(id: String)
+trait BaseBag {
+  val info: BagInfo
```

Comment:
We've called it data in the works, not sure if this is taken from the storage vocab though, in which case it feels like following that make sense.

Reply:
Yes, bag info is what it's called in the storage world I think